dockstore-integration-testing/src/test/java/io/dockstore/client/cli/BaseIT.java

```diff
@@ -74,6 +75,21 @@ public static void dropAndRecreateDB() throws Exception {
         CommonTestUtilities.dropAndRecreateNoTestData(SUPPORT);
         SUPPORT.before();
         testingPostgres = new TestingPostgres(SUPPORT);
+
+        // this is not great, doing this per test suite sucks. But at this point, there is no easy way to get the test tokens for both test users without loading both databases and dumping them too?
```

Comment:
I think you explained this on our Zoom, but to be clear, commenting this out only affects the rate limit reporting?

Reply:
I'm going to pick up from here for https://ucsc-cgl.atlassian.net/browse/SEAB-2519 but wanted this around. 
This would have added a new way to measure rate limit by test suite